### PR TITLE
Test/gameover tests

### PR DESCRIPTION
**Pull request by lilianaweimer**
## Pull request checklist
Please check if your PR fulfills the following requirements:
- [ ] Lint (`npm run lint`) has passed locally and any fixes were made for failures
I tried to run the linter and it didn't work, not sure why. I think the error was that it couldn't find the module `lint`.

## Pull request type
 
 
Please check the type of change your PR introduces:
- [ ] Bugfix
- [ ] Feature
- [x] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [x] Testing
- [ ] Other (please describe): 

## What is the current behavior?

- Added a lot of tests, and fixed an indentation error from my editor

## What is the new behavior?

- Added tests for: `GameOver`, `Header`, and `Lives`
- Fixed an indentation issue in `Form.test`

## Does this introduce a breaking change?
- [ ] Yes
- [x] No

### CODE REVIEW

File: src/Chalkboard/Chalkboard.test.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+      //               <Chalkboard />
+      //           </Provider>
+      //       </BrowserRouter>);
+        
+
+      // const question = getByText('Question');
+
+      // expect(question).toBeInTheDocument();
+		})
+
```
Comment (KyleWong2510) [inline src/Chalkboard/Chalkboard.test.js:27]: we can work on this one later

File: src/GameOver/GameOver.test.js [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+					<Provider store={store}>
+							<GameOver />
+					</Provider>
+			</BrowserRouter>);
+
+			const lives = getByText('Lives Left: 1');
+			
+			expect(lives).toBeInTheDocument();
+	});
+
```
Comment (KyleWong2510) [inline src/GameOver/GameOver.test.js:54]: Nice tests.  Good job passing only what you need into the mock store.

File: src/Header/Header.test.js [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+    //also, the lives live in their own element (the actual #)
+    //so they need to be tested for separately 
+
+    expect(question).toBeInTheDocument();
+    expect(round).toBeInTheDocument();
+    expect(lives).toBeInTheDocument();
+    expect(score).toBeInTheDocument();
+	}); 
+
+});
```
Comment (KyleWong2510) [inline src/Header/Header.test.js:45]: We will get the tests working for the other components

File: src/GameOver/GameOver.test.js [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+import GameOver from './GameOver';
+import thunk from 'redux-thunk';
+import { Provider } from 'react-redux';
+import { createStore } from 'redux';
+
+const store = createStore(() => ({setScore: 10, setLives: 1}))
+
+describe('Game Over', () => {
+
+	it('should render without crashing', () => {
```
Comment (Billwilke42) [inline src/GameOver/GameOver.test.js:15]: Nice job!

File: src/Header/Header.test.js [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+import { createStore } from 'redux';
+
+const store = createStore(() => ({setScore: 10, setLives: 1}))
+
+describe('Header', () => {
+
+	it('should render without crashing', () => {
+		const { getByText } = render(
+				<BrowserRouter>
+						<Provider store={store}>
```
Comment (Billwilke42) [inline src/Header/Header.test.js:18]: I can fix the tabbing errors

File: src/Lives/Lives.test.js [hunk 6]
```diff
@@ -0,0 +1,35 @@
...
+              currentRound={3}
+              score={4}
+            />
+						</Provider>
+        </BrowserRouter>);
+        
+    const lives = getByText('2');
+
+    expect(lives).toBeInTheDocument();
+	});
```
Comment (KyleWong2510) [inline src/Lives/Lives.test.js:33]: Good job.  We need to figure out your tabbing!

**Pull request merged**